File: Wirtualny_Swiat_Java/Projekt_PO_Java/src/model/Organisms/Wolf.java
package model.Organisms;

import model.Organism;
import model.Point;
import model.World;
import model.Animal;

import java.awt.*;

public class Wolf extends Animal {
    private Color wilkColor = Color.GRAY;

    public Wolf(World world, Point position){
        super(world, position, 9, 5);
    }

    @Override
    public Color getColor(){
        return wilkColor;
    }

    @Override
    public String getNazwa(){
        return "Wilk";
    }

    @Override
    public Organism makeNewOrganism(Point position){
        return new Wolf(world, position);
    }

}

File: WirtualnySwiatPython/Animals/Wolf.py
from animal import Animal
from config import GRAY

class Wolf(Animal):
    def __init__(self, world, position):
        super().__init__(world, position, power=9, initiative=5)
        self.color = GRAY

    def getColor(self):
        return self.color
    def getNazwa(self):
        return "Wilk"